wps-cloudformation-template.yaml

```diff
@@ -11,6 +11,8 @@ Parameters:
     Type: String
   geonetworkCatalogueURL:
     Type: String
+  sourceArn:
```

Comment:
This should be an optional parameter, so defaulting to '' so that things like dev environments don't have to include it.

Reply:
Default added